SPECS-SIGNED/kernel-signed-aarch64/kernel-signed-aarch64.spec

```diff
@@ -51,7 +46,7 @@ mkdir -p %{buildroot}/%{_localstatedir}/lib/initramfs/kernel
 
 cp -rp ./boot/* %{buildroot}/boot
 cp -rp ./lib/* %{buildroot}/lib
-cp -rp ./var/* %{buildroot}/%{_localstatedir}
+cp -rp .%{_var}/* %{buildroot}/%{_localstatedir}
```

Comment:
I'd argue that this is an issue with the linter and not the spec. `%{_var}` is tied to how files are organized on the final system and here we're accessing the source's internal directory, so it's not the same thing/purpose. Not to mention, that if we ever change `${_var}` (doubtful) we will break this spec.

Btw. I believe linter's config is still being tweaked, so feel free to report any suggestions to @thcrain-msft.

Reply:
@PawelWMS Thanks for pointing this out. Restored the ./var value. Will follow up on the linter side